.config/dotnet-tools.json

```diff
@@ -10,7 +10,7 @@
       "rollForward": false
     },
     "swashbuckle.aspnetcore.cli": {
-      "version": "6.5.0",
+      "version": "6.6.2",

```

Comment:
@DavidJGapCR  Should we be using version 8.x instead of 6.x?

Reply:
That's weird. I think that page is out of date, given that version 9 is out, and I've now confirmed that [version 9 supports .net 8](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/releases) (whereas previously I assumed it would only support .net 9).

Reply:
My concern with the old version is that it might not work if you don't have the 6.0 runtime or SDK, so might as well be on the latest version. But this isn't a showstopper. The PR can move forward as is.